tests/snapshots/tests.contrib.django.test_django_snapshots.test_middleware_trace_partial_based_view_21x.snap

```diff
@@ -0,0 +1,246 @@
+                                   "error" 0
+                                   "span_id" 19
+                                   "trace_id" 0
+                                   "parent_id" 16
+                                   "start" 1603387534004026729
+                                   "duration" 729924}
+                                      {"name" "django.middleware"
+                                       "service" "django"
+                                       "resource" "tests.contrib.django.middleware.EverythingMiddleware"
+                                       "error" 0
+                                       "span_id" 21
+                                       "trace_id" 0
+                                       "parent_id" 19
+                                       "start" 1603387534004314394
+                                       "duration" 436843}
+                                          {"name" "django.middleware"
+                                           "service" "django"
+                                           "resource" "tests.contrib.django.middleware.EverythingMiddleware.__call__"
+                                           "error" 0
+                                           "span_id" 22
+                                           "trace_id" 0
+                                           "parent_id" 21
+                                           "start" 1603387534004417487
+                                           "duration" 328008}
+                                              {"name" "django.middleware"
+                                               "service" "django"
+                                               "resource" "django.middleware.csrf.CsrfViewMiddleware.process_view"
+                                               "error" 0
+                                               "span_id" 23
+                                               "trace_id" 0
+                                               "parent_id" 22
+                                               "start" 1603387534004548257
+                                               "duration" 17657}
+                                              {"name" "django.middleware"
+                                               "service" "django"
+                                               "resource" "tests.contrib.django.middleware.EverythingMiddleware.process_view"
+                                               "error" 0
+                                               "span_id" 24
+                                               "trace_id" 0
+                                               "parent_id" 22
+                                               "start" 1603387534004599101
+                                               "duration" 11020}
+                                              {"name" "django.view"
+                                               "service" "django"
+                                               "resource" "partial"
```

Comment:
This resource name isn't particularly useful. We should probably follow up addressing this.